main.go

```diff
@@ -108,26 +212,60 @@ func main() {
 	}
 	filePath := os.Args[1]
 	debug("## Loading " + filePath)
-
-	fileData, err := ioutil.ReadFile(filePath)
+	
+	var subsetPartition map[int][]int
```

Comment:
This line meant to be deleted? Doesn't seem necessary given the `err2, subsetPartition := partition(test.Config)` below.